site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -278,15 +281,15 @@ To remove a curve simply blank all boxes."></i>
                     <legend><h2>Display Benchmarks</h2></legend>
                     {% for benchmark in display_benchmarks %}
                         <p>
-                            <input type="checkbox" id="display_benchmarks_{{ benchmark.id }}" name="display_benchmarks" value="{{ benchmark.id }}" data-testid="display-benchmarks-{{ benchmark.id }}"
+                            <input type="checkbox" id="display_benchmarks_{{ benchmark.id }}" name="display_benchmarks" value="{{ benchmark.id }}" data-testid="display_benchmarks_{{ benchmark.id }}"
```

Comment:
This should stay as hyphens

Reply:
[55558e2](https://github.com/Submitty/Submitty/pull/10726/commits/55558e24450f2262363a9376c9acb2310981c59a)